components/tabzilla.jsx

```diff
@@ -0,0 +1,64 @@
+// This is a Tabzilla "lazy-loader" for sites that don't use jQuery.
+// Because Tabzilla requires jQuery and jQuery is relatively large,
+// we delay loading it until the last possible moment, when the user
+// has expressed intent to explore Tabzilla.
+//
+// Only one instance of this component is expected to ever be on the
+// page at once, and it's expected to exist for the lifetime of the
+// whole page.
+//
+// This component does require the following stylesheet to be loaded,
+// however:
+//
+//   //mozorg.cdn.mozilla.net/media/css/tabzilla-min.css
+
+var React = require('react');
+
+// This is currently the minimum version of jQuery that Tabzilla requires
+// to load.
+var JQUERY = '//mozorg.cdn.mozilla.net/media/js/libs/jquery-1.7.1.min.js';
```

Comment:
We may want to rely on https://api.jquery.com/jquery.noconflict/ here so that we can at least control what happens should there be a jQuery conflict.